R4ML/R/ml.model.cox.R

```diff
@@ -311,7 +311,7 @@ setMethod(f = "show", signature = "r4ml.coxph", definition =
 #' @name summary.r4ml.coxph
 #' @title Cox Summary
 #' @description Summarizes cox proportional hazard analysis
-#' @param object A cox model \code{r4ml.matrix}
+#' @param object (r4ml.matrix):  A cox model
```

Comment:
sorry for late reply. but isn't r4ml.coxph is a matrix? I mean from the data type perspective?